src/app/common/utils/common.utils.random.ts

```diff
@@ -1,3 +1,7 @@
 export function getRandomNumber(min: number, max: number): number {
   return Math.random() * (max - min) + min;
 }
+
+export function getRandomNumberFloor(min: number, max: number): number {
+  return Math.floor(getRandomNumber(min, max));
```

Comment:
It should not be rounded instead? And function name change to `getRoundedRandomNumber`?